compiler/pash_compilation_server.py

```diff
@@ -414,6 +429,16 @@ def run(self):
             self.parse_and_run_cmd(input_cmd)
         
         self.connection_manager.close()
+        if not self.parallelized_flag:
+            log("No parts of the input script were parallelized. Ensure commands are annotated for parallelization.")
+        elif all(
+            proc_info.exec_time is not None and proc_info.exec_time < 1
```

Comment:
Have we checked that this ever actually passes? I am skeptical about the all proc_info.exec_time is not None. Is there anyway one of those is None and this becomes false?